magicbot/magicrobot.py

```diff
@@ -17,6 +17,21 @@
 
 __all__ = ['MagicRobot']
 
+def feedback(key=None):
```

Comment:
No docstring? What is this supposed to do? Maybe add an example in the docstring too?